project_resource_calendar/models/calendar_event.py

```diff
@@ -65,6 +65,10 @@ class CalendarEvent(models.Model):
                           [('id', '!=', False), ('recurrency', '=', True), ('is_task_event', '=', True)]
                           )]}
     )
+    client_type = fields.Many2one(
```

Comment:
Better to define this field in project_event/model/calendar_event.py (where you added sector_id)